.ci/jobs/defaults.yml

```diff
@@ -1,23 +0,0 @@
-
```

Comment:
It looks like this file should be removed in a following PR.

In a first Pull Request, all files except `.ci/defaults.yml` should be removed. That would delete the jobs from the server.

In a second Pull Request, remove the whole `.ci` folder

@jlind23